module-1/README.md

```diff
@@ -69,67 +70,101 @@ In the terminal, change directory to the newly cloned repository directory:
-Next, we will create the infrastructure components needed for hosting a static website in Amazon S3 via the [AWS CLI](https://aws.amazon.com/cli/).
+#### Create an S3 Bucket for Storing Content
+Next, we will create the infrastructure components needed for storing static website in Amazon S3 via the ![AWS CLI](https://aws.amazon.com/cli/). If you do not already have the AWS CLI configured see ![getting started](http://docs.aws.amazon.com/cli/latest/userguide/)
 
-**Note: This workshop uses placeholders for names that you must supply. These placeholders use the prefix `REPLACE_ME_` to make them easy to find using CTRL-F on Windows or ⌘-F on Mac.**
-
-First, create an [S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html), replace *REPLACE_ME_BUCKET_NAME* with your own unique bucket name, as described in [requirements for bucket names](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html#bucketnamingrules).** Copy the name you choose and save it for later, as you will use it in several other places during this workshop:
+First, create an S3 bucket, replace the name below (mythical-mysfits-bucket-name) with your own unique bucket name.  Copy the name you choose and save it for later, as you will use it in several other places during this workshop:
 
 ```
-aws s3 mb s3://REPLACE_ME_BUCKET_NAME
+aws s3 mb s3://mythical-mysfits-bucket-name
 ```
 
-Now that we have created a bucket, we need to set some configuration options that enable the bucket to be used for [static website hosting](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html).  This configuration enables the objects in the bucket to be requested using a registered public DNS name for the bucket, as well as direct site requests to the base path of the DNS name to a selected website homepage (index.html in most cases):
+#### Update the S3 Bucket Policy
+
+All buckets created in Amazon S3 are fully private by default.  In order to be used as a website accessible only through CloudFront, we need to create an S3 [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html) that indicates objects stored within this new bucket may be accessed by a CloudFront origin access identity that we also create. Bucket policies are represented as JSON documents that define the S3 *Actions* (S3 API calls) that are allowed (or explicitly denied) to be performed by different *Principals* (in our case the CloudFront service).
+
+First we need to create a CloudFront Access Identity using the following CLI command.
 
 ```
-aws s3 website s3://REPLACE_ME_BUCKET_NAME --index-document index.html
+aws cloudfront create-cloud-front-origin-access-identity --cloud-front-origin-access-identity-config CallerReference=Mysfits,Comment=Mysfits
 ```
 
-#### Update the S3 Bucket Policy
+Record the **Id** from the output as we will need to update the bucket policy with this.
+
+```
```

Comment:
I don't think you need this inline bucket policy here and it might confuse attendees who could assume this instruction is saying this code block is something they need to copy and use directly as the bucket policy (if they already know how to edit bucket policies).

Reply:
Added label for the example output, if this is still confusing we can remove code block completely.